openzaak/src/main/kotlin/com/ritense/openzaak/form/OpenZaakFormFieldDataResolver.kt

```diff
@@ -23,13 +23,13 @@ import com.ritense.openzaak.service.impl.ZaakService
 import com.ritense.valtimo.contract.form.DataResolvingContext
 import com.ritense.valtimo.contract.form.ExternalFormFieldType
 import com.ritense.valtimo.contract.form.FormFieldDataResolver
-import java.util.UUID
 
 class OpenZaakFormFieldDataResolver(
     private val zaakService: ZaakService,
     private val zaakInstanceLinkService: ZaakInstanceLinkService
 ) : FormFieldDataResolver {
 
+    @Deprecated("Deprecated in Java")
```

Comment:
I don't fully get what the message is supposed to say. It will be removed for java but will still be available for Kotlin?

The @Deprecated should also include the "since" attribute telling in what version the method became deprecated so we know when we can remove it

Reply:
It's a new kotlin thing where you are required to mark methods as deprecated when the method is deprecated in the class or interface that defines it. I could change this, but it should be removed in the next story anyway.

Reply:
ok didn't know that. Thanks for the clarification